tuplex/python/include/PythonDataSet.h

```diff
@@ -77,8 +77,9 @@ namespace tuplex {
         PythonDataSet resolve(const int64_t exceptionCode, const std::string& lambda_code, const std::string& pickled_code, const py::object& closure=py::object());
 
         py::object collect();
-        py::object take(const int64_t numRows);
+        py::object take(const int64_t topLimit, const int64_t bottomLimit);
         void show(const int64_t numRows=-1);
+        std::string showHTMLPreview(const int64_t topLimit, const int64_t bottomLimit);
```

Comment:
if we put the logic of producing the HTML table into the python API we could save this double function. I.e., simply call from python api take with the correct arguments and turn then everything into a table?

Other option is to call this function takeWithLimits or so.